### PR TITLE
Ardupilot downstream 20200129

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -5671,6 +5674,9 @@
       <description>Status text message. These messages are printed in yellow in the COMM console of QGroundControl. WARNING: They consume quite some bandwidth, so use only for important status and error messages. If implemented wisely, these messages are buffered on the MCU and sent only at a limited rate (e.g. 10 Hz).</description>
       <field type="uint8_t" name="severity" enum="MAV_SEVERITY">Severity of status. Relies on the definitions within RFC-5424.</field>
       <field type="char[50]" name="text">Status text message, without null termination character</field>
+      <extensions/>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: This one is being dealt with in https://github.com/mavlink/mavlink/pull/1295

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -5169,6 +5170,8 @@
       <field type="uint8_t" name="satellites_visible">Number of satellites visible. If unknown, set to 255</field>
       <field type="uint8_t" name="dgps_numch">Number of DGPS satellites</field>
       <field type="uint32_t" name="dgps_age" units="ms">Age of DGPS info</field>
+      <extensions/>
+      <field type="uint16_t" name="yaw" units="cdeg">Yaw in earth frame from north. Use 0 if this GPS does not provide yaw. Use 65535 if this GPS is configured to provide yaw and is currently unable to provide it. Use 36000 for north.</field>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: Why not make a float and use NaN for "not supported"? Or use an int and use -1 for "not supported". Just efficiency?

@julianoes Are you OK with this?

**Comment by hamishwillee** [comment]
Thanks @amilcarlucas. I've added a few comments. Generally it would be nice if ArduPilot could submit changes to common into this repo first. It is not too much to ask.

File: message_definitions/v1.0/common.xml (hunk 2 above)
Comment (hamishwillee) [reply message_definitions/v1.0/common.xml]: @julianoes ?

Comment (auturgy) [reply message_definitions/v1.0/common.xml]: 0 for not supported means it gets trimmed.  A NaN or -1 would not.

Comment (hamishwillee) [reply message_definitions/v1.0/common.xml]: Good point. Well, everyone has had plenty of opportunity to comment and this breaks nothing by default, so I'm going to merge.

Comment (julianoes) [reply message_definitions/v1.0/common.xml]: Fine with me. Sorry for not responding.

**Pull request merged**